### ISSUE TITLE
Transform for trainset and valset during pretraining

### ISSUE THREAD

**Issue by hoangtnm** [facebookresearch/CovidPrognosis#20]
Hi, after reading transforms used for pretraining MoCo, its seems that trainset and valset use the [same transforms](https://github.com/facebookresearch/CovidPrognosis/blob/5a8731e86f263b77263ef5dee8faae9a4cc0d661/cp_examples/moco_pretrain/train_moco.py), which is
```python
    transform_list = [
        transforms.RandomResizedCrop(args.im_size, scale=(0.2, 1.0)),
        transforms.RandomHorizontalFlip(),
        transforms.RandomVerticalFlip(),
        transforms.ToTensor(),
        RandomGaussianBlur(),
        AddGaussianNoise(snr_range=(4, 8)),
        HistogramNormalize(),
        TensorToRGB(),
    ]
```
however, the [transform implemented in MoCoV2](https://github.com/PyTorchLightning/lightning-bolts/blob/master/pl_bolts/models/self_supervised/moco/transforms.py#L165) seems to avoid non-deterministic transforms such as `Random*`. So I think there are some differences between the original MoCoV2 and this paper implementation. Do you have any idea about this one?

**Comment by mmuckley** [comment]
Hello @hoangtnm, changing it to deterministic is fine I think. It would make the loss monitoring a little more consistent, but I doubt you would see any major changes.
